google-cloud-spanner/acceptance/spanner_helper.rb

```diff
@@ -140,34 +140,32 @@ def assert_commit_response resp, commit_options = {}
 db_job.wait_until_done!
 raise GRPC::BadStatus.new(db_job.error.code, db_job.error.message) if db_job.error?
 
-unless emulator_enabled?
-  instance_path = $spanner_db_admin.instance_path project: $spanner.project_id, instance: $spanner_instance_id
-  db_job = $spanner_db_admin.create_database parent: instance_path,
+instance_path = $spanner_db_admin.instance_path project: $spanner.project_id, instance: $spanner_instance_id
+db_job = $spanner_db_admin.create_database parent: instance_path,
```

Comment:
Please adjust the indentation of the following two lines to match the change to this line. (Corrects rubocop failure.)

Reply:
Fixed.